app/controllers/result_filters/provider_controller.rb

```diff
@@ -0,0 +1,29 @@
+module ResultFilters
+  class ProviderController < ApplicationController
+    include FilterParameters
+
+    def new
+      if params[:query].blank?
+        flash[:error] = "Training provider"
```

Comment:
The error at the top of the page looks like this:

<img width="637" alt="image" src="https://user-images.githubusercontent.com/15608/73671989-fdce7100-46a3-11ea-99c2-b44ff2ae8e49.png">

Even though the error above the input box looks correct:

<img width="640" alt="image" src="https://user-images.githubusercontent.com/15608/73672050-176fb880-46a4-11ea-9a8c-1be092b1b748.png">

Reply:
Copied csharp feature

Reply:
Just double checked this with Misha and it seems to be correct

Reply:
Just having a chat with @aliuk2012 and I'm not so sure now @faissalMT, sorry. I'll let Al add his $0.02.

Reply:
Ideally, The name in error summary (top part of the page) and the error message near the invalid form field should be identical.

If I were a user I would wonder whether there were two errors and where to correct the second one. 

As a side note note I also spotted that there is a missing red left border across all the radio buttons like this:
![image](https://user-images.githubusercontent.com/3441519/73674569-b8f90900-46a8-11ea-9c53-d2e4e75b3548.png)

Perhaps something we could fix that in another PR.


For extra reference: https://design-system.service.gov.uk/components/error-message/

Reply:
🤦‍♂ more noticeable, the input itself is not highlighted. I didn't spot that until I revisited this comment

Reply:
Since this is present in Find I propose putting it in a seperate PR